forge/src/main/java/org/openjdk/skara/forge/gitlab/GitLabRepository.java

```diff
@@ -114,7 +114,7 @@ public PullRequest createPullRequest(HostedRepository target,
         var pr = request.post("merge_requests")
                         .body("source_branch", sourceRef)
                         .body("target_branch", targetRef)
-                        .body("title", draft ? "WIP: " : "" + title)
+                        .body("title", (draft ? "Draft: " : "") + title)
```

Comment:
During my test, I was surprised that when `draft` is true, the title will be "Draft: " without the real title. Seems like priority of operators is a little weird. `+` is greater than `?:`.

Reply:
I think `?:` has one of the lowest precedents in C-like languages. If you think of it more like an "if" construct, it kind of make sense. But nicely spotted, anyway!

Reply:
Nice found.

Reply:
> I think `?:` has one of the lowest precedents in C-like languages. If you think of it more like an "if" construct, it kind of make sense. But nicely spotted, anyway!

Make sense! Thx!